api/samples/opcode_count.cpp

```diff
@@ -101,6 +101,15 @@ event_opcode_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *i
     return DR_EMIT_DEFAULT;
 }
 
+static dr_emit_flags_t
+event_bb_analysis(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
+                  bool translating, OUT void **user_data)
+{
+    intptr_t bb_size = (intptr_t)drx_instrlist_app_size(bb);
```

Comment:
Hmm, `ptr_int_t` is what DR defines and uses.  Looks like there are existing uses of `intptr_t`...mostly from you :)  I guess it's the more standard name externally in stdint.h so for a sample it seems reasonable.

Reply:
Would it be worthwhile to do the switch to intptr_t? Don't really have a strong opinion tbh.

Reply:
^ I mean throughout the DR code base - of course via a separate issue.

Reply:
^ I mean throughout the DR code base - of course via a separate issue.

Reply:
No I would leave core DR with its own types.  IIRC stdint.h was *not* part of Visual Studio until rather recently which was part of the reason to have our own types.